packages/generate-pdf/src/index.ts

```diff
@@ -241,9 +257,15 @@ export async function generatePdf(
   options: GeneratePdfOptions = {},
 ): Promise<Buffer | PDFKit.PDFDocument> {
   const opts: GeneratePdfOptions = options ? { ...generatePdfOptions, ...options } : generatePdfOptions;
+
+  if (opts.attachCertificate && opts.outputType === 'stream') {
+    throw new Error('Cannot attach certificate to a stream output type. Please use buffer output type.');
+  }
```

Comment:
Even though this might not be very performant it is still possible to apply the attachCertificateToPdf and return a readable stream out of the buffer; hint Readable.from(YourBuffer).

Reply:
Yes, but I do not want to encourage that. Just use a buffer if you want this feature. The conversion should be done on the caller side if a stream is required.

Reply:
We have decided to remove the 'stream' output type all together.